3.5-windowsservercore-1709/runtime/Dockerfile

```diff
@@ -21,12 +21,12 @@ RUN powershell -Command `
         $ProgressPreference = 'SilentlyContinue'; `
         Invoke-WebRequest `
             -UseBasicParsing `
-            -Uri "http://download.windowsupdate.com/c/msdownload/update/software/secu/2018/02/windows10.0-kb4074588-x64_baac70613d5503dcd3e652ac24b7f432f3579eef.msu" `
+            -Uri "http://download.windowsupdate.com/d/msdownload/update/software/secu/2018/03/windows10.0-kb4088776-x64_55756340f1e2c2090f94de6d256eafd75e1cee9c.msu" `
```

Comment:
It's interesting that this one goes from `download.windowsupdate.com/c` to `...d`, and the other one goes from `d` to `c`. Anything to this, or just a coincidence?

Reply:
I don't think so.  They toggle back and forth periodically - (see https://github.com/MichaelSimons/dotnet-framework-docker/commit/d4ba37e1a9f85006c66358aa5b9ed094636ba3d9#diff-6c58c0038f638807db04ea974532ae7b).  FWIW, the urls are retrieved from http://www.catalog.update.microsoft.com/Search.aspx?q=1709 and http://www.catalog.update.microsoft.com/Search.aspx?q=1607